src/server.h

```diff
@@ -295,6 +295,9 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 
 	inline void setAsyncFatalError(const std::string &error)
 			{ m_async_fatal_error.set(error); }
+	inline void setAsyncFatalError(const LuaError &e) {
```

Comment:
Either  `{ etAsyncFatalError(std::string("Lua: ") + e.what()); }`or a line break for the bracket.